apps/core/scenes/results/EmptyResults.js

```diff
@@ -1,13 +1,13 @@
 // @flow
 
 import * as React from 'react';
-import { View } from 'react-native';
-import { Text } from '@kiwicom/universal-components';
+import { IllustrationWithInformation } from '@kiwicom/margarita-components';
 
 export default function EmptyResults() {
   return (
-    <View>
-      <Text>EmptyResults</Text>
-    </View>
+    <IllustrationWithInformation
+      illustrationName="NoResults"
+      text="Sorry, we couldn't find that connection."
```

Comment:
Maybe we can also add `Try selecting different dates or nearby places.` suggestion like in web version. But it's totally up to you, I'm ok even with current state.

<img width="349" alt="screenshot 2019-01-22 at 14 15 58" src="https://user-images.githubusercontent.com/2660330/51541263-aecaea80-1e58-11e9-83dd-fd489f487599.png">

Reply:
Done (I changed the text according to new one in Figma)

Reply:
<img width="359" alt="screenshot 2019-01-22 at 16 06 40" src="https://user-images.githubusercontent.com/858321/51544225-bb067600-1e5f-11e9-854d-c556173413b1.png">